docs/configuration/device-config/lora.mdx

```diff
@@ -25,7 +29,27 @@ Sets the region for your node. Default is `unset`. As long as this is not set, t
 
 ### Modem Preset
 
-Use a sensible default for the modem. Default is `unset` which equates to `LONG_FAST`. Also 'Use Presets' has to be defined, which is the default. If 'Use Presets' is not defined, the modem will be configured manually with Bandwidth, Spread Factor, and Coding Rate.
+Default is `unset` which equates to `LONG_FAST`. Presets are pre-defined modem settings (Bandwidth, Spread Factor, and Coding Rate) which influence both message speed and range. The default will provide a strong mixture of speed and range, for most users.
+
+The presets are designed to provide further options for optimizing either speed (and reduced network congestion) or range, which can be useful for two real world scenarios:
+
+1. A high number of devices exist in the mesh, or messages are sent very frequently. Faster speeds (and therefore lower radio time per device) can help with mesh network congestion.
+2. Maximum range is desired, for long range scenarios where a several second delay in message receipt is acceptable (for instance, attempting to send messages from a town to a distant mountain top).
+
+The Presets available as as follows, and follow a linear pattern of Fastest <--> Slowest, and Shortest <--> Longest range:
+
+1. `SHORT_FAST` (Fastest, highest bandwidth, lowest airtime, shortest range)
+
+2. `SHORT_SLOW`
+
+3. `MEDIUM_FAST`
+
+4. `MEDIUM_SLOW`
+
+5. `LONG_FAST` (Default)
+
+6. `LONG_SLOW`
+7. `VERY_LONG_SLOW` (Slowest, lowest bandwidth, highest airtime, longest range)
```

Comment:
Looks great! One thing we may consider is mentioning device count limits here. But we can do that another day.